Cargo.toml

```diff
@@ -225,6 +225,7 @@ getset = "0.1.3"
 rrs-lib = "0.1.0"
 rand = { version = "0.8.5", default-features = false }
 hex = { version = "0.4.3", default-features = false }
+serde_arrays = "0.1.0"
```

Comment:
is this different from https://crates.io/crates/serde-big-array ? 
we were already using that one in some macros, and it seems more maintained (and originally suggested by dtolnay which is a rust expert)